### PR TITLE
[DOC] Fix broken links in doc for Tempo v1.2

### PR DESCRIPTION
**Pull request by knylander-grafana**
Fixes two broken links in Tempo docs for v1.2. This is part of an effort to remove all broken links from all documentation. 

**Which issue(s) this PR fixes**:
Part of: https://github.com/grafana/technical-documentation/issues/811

### CODE REVIEW

**Review by zalegrala — APPROVED** [review]

**Comment by knylander-grafana** [comment]
Local test shows the links are fixed.

File: docs/README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Tempo Documentation
 
-<p align="center"> <img src="tempo/website/logo_and_name.png" alt="Tempo Logo"> <br>
+<p align="center"> <img src="sources/tempo/logo_and_name.png" alt="Tempo Logo"> <br>
   
 Please see the [Documentation Site](https://grafana.com/docs/tempo/latest/) The files in this directory are used to generate it but consequently the links don't work in Github.
```
Comment (knylander-grafana) [inline docs/README.md:5]:
```suggestion
Please see the [Documentation Site](https://grafana.com/docs/tempo/latest/). The files in this directory are used to generate it but consequently the links don't work in Github.
```

**Review by jdbaldry — APPROVED** [review]
Thanks for checking the links with `make docs` :)

File: docs/tempo/website/api_docs/_index.md [hunk 2]
```diff
@@ -82,11 +82,11 @@ Agent / OpenTelemetry Collector / Jaeger Agent.
 |  Jaeger | GRPC | [Link](https://www.jaegertracing.io/docs/latest/apis/#span-reporting-apis) |
 |  Zipkin | HTTP | [Link](https://zipkin.io/zipkin-api/) |
 
-_For information on how to use the Zipkin endpoint with curl (for debugging purposes) check [here](pushing-spans-with-http)._ 
+_For information on how to use the Zipkin endpoint with curl (for debugging purposes) check [here](pushing-spans-with-http)._
```
[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/tempo/website/api_docs/_index.md:85]: Checking now.

[the remark this answers is not in the document]
Comment (knylander-grafana) [reply docs/tempo/website/api_docs/_index.md:85]: Link works.

**Pull request merged**